frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,24 @@
+import { Navigate } from 'react-router-dom';
```

Comment:
And add reexport of it on `libs/components/components`
```suggestion
import { Navigate } from '~/libs/components/components.js';
```